ghcide/src/Development/IDE/Plugin/CodeAction.hs

```diff
@@ -1090,10 +1090,10 @@ constructNewImportSuggestions
 constructNewImportSuggestions exportsMap (qual, thingMissing) notTheseModules = nubOrd
   [ suggestion
   | Just name <- [T.stripPrefix (maybe "" (<> ".") qual) $ notInScope thingMissing]
-  , (identInfo, m) <- maybe [] Set.toList $ Map.lookup name (getExportsMap exportsMap)
+  , identInfo <- maybe [] Set.toList $ Map.lookup name (getExportsMap exportsMap)
   , canUseIdent thingMissing identInfo
-  , m `notElem` fromMaybe [] notTheseModules
-  , suggestion <- renderNewImport identInfo m
+  , moduleNameText identInfo `notElem` fromMaybe [] notTheseModules
+  , suggestion <- renderNewImport identInfo (moduleNameText identInfo)
```

Comment:
Do we need to pass `(moduleNameText identInfo)` part? That's now redundant as it is in `identInfo`.

Reply:
well spotted, thanks